api/python/tests/test_util.py

```diff
@@ -55,13 +55,7 @@ def test_read_yaml(tmpdir):
     assert parsed_string == parsed_path_obj
 
 
-def test_yaml_has_comments(tmpdir):
```

Comment:
Here again, we should fix this test or, if comments are not supported, then use `pytest.raises`.

Reply:
The reason why I had this test removed is that the underlying [function](https://github.com/quiltdata/quilt/blob/d7508fb2ad6513a5c34b7ad5c57c29c176b33925/api/python/quilt3/util.py#L279) this test calls to check if yaml has comments is written specifically to test for comment objects in `ruamel CommentedBase` and `pyyaml` has no such functionality which makes the function useless hence why I removed the function and the test.